fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/BinaryObject.java

```diff
@@ -47,7 +47,7 @@ protected BinaryObject(PID pid, Repository repository, RepositoryObjectDataLoade
 		super(pid, repository, dataLoader);
 		
 		metadataUri = URI.create(pid.getRepositoryUri().toString()
-				+ "/" + RepositoryPathConstants.FCR_METADATA);
+				+ RepositoryPathConstants.FCR_METADATA);
```

Comment:
I made the change so that the unit test would pass. That's TDD if you ask me :-)